identity-account/src/account/account.rs

```diff
@@ -47,7 +47,7 @@ pub struct Account {
   actions: AtomicUsize,
   chain_state: ChainState,
   state: IdentityState,
-  did_lease: DIDLease,
+  _did_lease: DIDLease,
```

Comment:
We probably need a comment to explain that although `did_lease` is not used in the `Account`, it needs to stay since it has special behaviour when dropped, which is expected when the `Account` is dropped.

Reply:
I added this as a comment.